lib/rules/shorthand-property-no-redundant-values/index.js

```diff
@@ -35,10 +35,7 @@ function isShorthandProperty(property) {
 	return propertiesWithShorthandNotation.has(property);
 }
 
-function canCondense(top, right) {
-	const bottom = arguments.length > 2 && arguments[2] !== undefined ? arguments[2] : null;
-	const left = arguments.length > 3 && arguments[3] !== undefined ? arguments[3] : null;
-
+function canCondense(top, right, bottom, left) {
```

Comment:
Are you sure this is equivalent and cannot fail?

Reply:
Should be ok. I reviewed relevant code and it should work the same both with `null` and `undefined`.